src/logging.h

```diff
@@ -94,6 +98,15 @@ namespace BCLog {
         /** Log categories bitfield. */
         std::atomic<uint32_t> m_categories{0};
 
+        /** Category-specific threshold log level. This level overrides the global m_threshold_level. */
+        std::unordered_map<LogFlags, Level> m_category_levels;
+
+        /**
+         * If there is no category-specific threshold log level, all logs with severity level lower than
+         * m_threshold_level will be ignored.
+         */
+        Level m_threshold_level{Level::Info};
```

Comment:
I'd find this data member naming clearer as `m_severity_level` or `m_log_level` (I prefer the latter), maybe that is just me.